src/model/channel/message.rs

```diff
@@ -72,6 +72,13 @@ pub struct Message {
     /// Array of [`Role`]s' Ids mentioned in the message.
     pub mention_roles: Vec<RoleId>,
     /// Channels specifically mentioned in this message.
+    /// **Note**:
+    /// Not all channel mentions in a message will appear in mention_channels.
+    /// Only textual channels that are visible to everyone in a lurkable guild will ever be included.
+    /// Only crossposted messages (via Channel Following) currently include mention_channels at all.
+    /// If no mentions in the message meet these requirements, this field will not be sent.
+    /// [`See more in the discord documentation`].
+    /// https://discord.com/developers/docs/resources/channel#message-object
```

Comment:
You can see examples for reference-style links [here](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links)

```markdown
    /// 👇 This is the link text by itself
    /// [`See more in the discord documentation`].
    ///
    /// 👇 This is where the link is defined which needs a blank line before it.
    /// [`See more in the discord documentation`]: https://discord.com/developers/docs/resources/channel#message-object
```

```suggestion
    /// [`See more in the discord documentation`].
    ///
    /// [`See more in the discord documentation`]: https://discord.com/developers/docs/resources/channel#message-object
```